validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -82,6 +80,7 @@ def __str__(self):
         return out[:-1] + "}"
 
 
+# pylint: disable=stop-iteration-return
```

Comment:
Is this related to - https://github.com/PyCQA/pylint/issues/1830 ?  If so, might be worth a mention in the commit message or next to this comment.

Reply:
yeah